ui/src/stat_table.tsx

```diff
@@ -43,6 +43,8 @@ export interface StatTableItem {
   label: S
   /** The values displayed in the row. */
   values: S[]
+  /** List of font colors used for each value in values ordered respectively. Defaults to theme text color if not specified. */
+  value_colors?: S[]
```

Comment:
Always put new optional attributes at the end since some people may use positional args (not keyword args) in python. 

```suggestion
  /** List of colors used for each value in values ordered respectively. */
  colors?: S[]
```